packages/graphile-build-pg/src/plugins/makeProcField.js

```diff
@@ -629,6 +629,7 @@ export default function makeProcField(
               }
             )}`,
             isMutationInput: true,
+            ...payloadTypeScope,
```

Comment:
I do not like reusing the other scope here; please create a new scope for the input type, even if it ends up containing the same things.